libraries/chain/hotstuff/hotstuff.cpp

```diff
@@ -184,6 +184,18 @@ valid_quorum_certificate::valid_quorum_certificate(
       _weak_votes = vector_to_bitset(weak_votes);
 }
 
+// Strong beats weak. Break tie with highest accumulated weight.
+bool valid_quorum_certificate::operator<(const valid_quorum_certificate& other) const {
```

Comment:
Remove this comparison function as well. There is no clear way to consider one QC better than the other as a total order. And we don't need a total order to get a good enough implementation for `get_best_qc` for now.